Modified Activity Code List

jsh-me opened the pull request:
* Modify MainActivity Code
Add Login Session Code
* Modify SettingActivity Code
Modify ListView Code - Add Logout function
* When exit an app, Exit All Background.

jsh-me committed:
Modified Activity Code List

* Modify MainActivity Code
Add Login Session Code
* Modify SettingActivity Code
Modify ListView Code - Add Logout function
* When exit an app, Exit All Background.

Blankymunn3 reviewed (approved):
현재로써는 로그인 유지보다는 로그인정보 유지라고 표현하는게 맞는거같네요.
일단 load()를 이용해서 이전 데이터를 받아와서 그값을 이용하여 디비와 연결하는걸 수행하면 될것같네요.
그부분을 알고리즘으로 짜서 구현해보세요.

jsh-me committed:
Maintain Login Session Successfully

* Maintain Login Session Successfully
Before : If login, Use Only id,pw data Cookies
After : Compared id,pw value to Database value.
And AutoLogin Connected.
